documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -22,3 +24,4 @@
 * (IDETECT-3821) Detect will now capture and record failures of the Signature Scanner due to command lengths exceeding Windows limits. This can happen with certain folder structures when using the `detect.excluded.directories` property.
 * (IDETECT-3820) Introduced an enhanced approach to NuGet Inspector for handling different formats of the `project.json` file, ensuring compatibility with both old and new structures.
 * (IDETECT-4027) Resolved a problem with the npm CLI detector for npm versions 7 and later, which was causing only direct dependencies to be reported.
+* (IDETECT-3997) Fix for npm package JSON parse detector classifying components as rubygems instead of npmjs.
```

Comment:
Minor rewording suggestion:

* (IDETECT-3997) Resolved npm package JSON parse detector issue of classifying components as RubyGems instead of npmjs.